### PR TITLE
Carried LM-types PR and modified with new models

### PR DESCRIPTION
**Pull request by merveenoyan**
cc @lewtun I carried your former PR and modified with new models for text-generation task page

### CODE REVIEW

File: packages/tasks/src/text-generation/about.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ The most popular models for this task are GPT-based models, [Mistral](mistralai/
 
 These models are trained to learn the mapping between a pair of texts (e.g. translation from one language to another). The most popular variants of these models are [NLLB](facebook/nllb-200-distilled-600M), [FLAN-T5](https://huggingface.co/google/flan-t5-xxl), and [BART](https://huggingface.co/docs/transformers/model_doc/bart). Text-to-Text models are trained with multi-tasking capabilities, they can accomplish a wide range of tasks, including summarization, translation, and text classification.
 
+## Language Model Variants
+
+When it comes to text generation, the underlying language model can come in several types:
+
+- **Base models:** this refers plain language models like [Mistral 7B](mistralai/Mistral-7B-v0.1) and [Llama-2-70b](https://huggingface.co/meta-llama/Llama-2-70b-hf). These models are good for fine-tuning and few-shot prompting.
```
Comment (pcuenca) [inline packages/tasks/src/text-generation/about.md:39]:
```suggestion
- **Base models:** refers to plain language models like [Mistral 7B](mistralai/Mistral-7B-v0.1) and [Llama-2-70b](https://huggingface.co/meta-llama/Llama-2-70b-hf). These models are good for fine-tuning and few-shot prompting.
```

File: packages/tasks/src/text-generation/about.md [hunk 2]
```diff
@@ -32,6 +32,16 @@ The most popular models for this task are GPT-based models, [Mistral](mistralai/
...
+
+- **Human feedback models:** these models extend base and instruction-trained models by incorporating human feedback that rates the quality of the generated text according to criteria like [helpfulness, honesty, and harmlessness](https://arxiv.org/abs/2112.00861). The human feedback is then combined with an optimization technique like reinforcement learning to align the original model to be closer with human preferences. The overall methodology is often called [Reinforcement Learning from Human Feedback](https://huggingface.co/blog/rlhf) or RLHF for short. [Llama2-Chat](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) is an open-source model aligned through human feedback.
```
Comment (pcuenca) [inline packages/tasks/src/text-generation/about.md:43]:
```suggestion
- **Human feedback models:** these models extend base and instruction-trained models by incorporating human feedback that rates the quality of the generated text according to criteria like [helpfulness, honesty, and harmlessness](https://arxiv.org/abs/2112.00861). The human feedback is then combined with an optimization technique like reinforcement learning to align the original model to be closer with human preferences. The overall methodology is often called [Reinforcement Learning from Human Feedback](https://huggingface.co/blog/rlhf), or RLHF for short. [Llama2-Chat](https://huggingface.co/meta-llama/Llama-2-7b-chat-hf) is an open-source model aligned through human feedback.
```

**Review by pcuenca — COMMENTED** [review]
Nice!

**Review by osanseviero — APPROVED** [review]
Thanks!

When doing changes from another author, please co-author the commits (https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)

**Comment by merveenoyan** [comment]
@lewtun I wanted to add your commit signature email to merger notes coauthor section but couldn't find it in the notes of your PRs, can you give that through slack?

**Pull request merged**